lib/lightning_web/live/run_live/index.html.heex

```diff
@@ -707,13 +707,18 @@
                   data-entity="work_order_list"
                   class="bg-white divide-y divide-gray-200 text-gray-500"
                 >
-                  <%= for wo <- @page.entries do %>
+                  <%= for entry <- @page.entries do %>
```

Comment:
Isn't `for workorder <- @page.entries do` a bit more readable ? I mean a person reading this code would know that entries are workorders. What do you think ?